packages/frontend/src/commons/index.ts

```diff
@@ -51,6 +55,11 @@ export interface ChainData {
 
 export const FEE_UNIT = 1_000_000;
 
+export const BLACKLISTED_ADDRESSES: Address[] = [
```

Comment:
Why did we add this? I don't think it's used anywhere else.

Reply:
It was used in the distribution breakdown, before the changes, to hide the zero and dead addresses in the leaderboard, should we add this on the API level?

Reply:
We already do with the 0 address. It's fine if the dead address shows up, as it indicates an error in the backend distribution. I think we can remove this

Reply:
Addressed in 9b157fb30b55f767c13f77f37040bdf92dd7a7f2.